### DISCUSSION TITLE
Does it support multi-layer simulation

### DISCUSSION THREAD

**Discussion by ytipocmk631** [PedestrianDynamics/jupedsim#1396] [Q&A] [answered]
Hi:
I want to try simulating multiple floors, but I don't seem to find any API for this scenario in the documentation. How can I implement it? thanks

**Comment by chraibi** [comment]
No, JuPedSim does not support multiple floors. You'll have to pass agents from one floor to the other.

**Comment by Ozaq** [comment, marked as answer]
Hi @ytipocmk631 2.5D geometry is on our development roadmap but not yet supported. We hope to have added this until end of this year. 

In the time being you can instantiate one simulation per level and add/remove agents from the respective simulation when they would cross over. This requires a bit of coding in python and there are issues to address when there is limited space in the application target level.

If you have further questions feel free to ask.
